frontend/i18next.scanner/en/translation.json

```
@@ -431,5 +431,12 @@
   "You have reach your storage limit, you cannot add new files": "You have reach your storage limit, you cannot add new files",
   "Sign in": "Sign in",
   "Login:": "Login:",
-  "Password:": "Password:"
+  "Password:": "Password:",
+  "Favorite": "Favorite",
```

Since we use the British flag, should it be 
```suggestion
  "Favourite": "Favourite",
```
?

---

We use british english in Tracim? I wasn't aware :-)

---

Done.